dash/include/dash/memory/GlobUnitMem.h

```diff
@@ -0,0 +1,470 @@
+#ifndef DASH__GLOB_UNIT_MEM_H__INCLUDED
```

Comment:
Unfortunately you copied the content of the file istead of the file. So we lose the history.